packages/cloud-cognitive/src/components/ActionBar/ActionBarItem.js

```diff
@@ -27,8 +36,8 @@ export let ActionBarItem = ({ inOverflow, ...otherProps }) => {
         hasIconOnly: true,
         kind: 'ghost',
         size: 'field',
-        tooltipPosition: inOverflow ? 'left' : 'bottom',
-        tooltipAlignment: inOverflow ? 'center' : 'end',
+        tooltipPosition: 'bottom',
+        tooltipAlignment: 'end',
         type: 'button',
       }}></Button>
```

Comment:
This can be changed to a self closing tag which will look/feel a bit more logical being that it is an icon only button like this...
```js
<Button
    someProp={value}
/>
```

Reply:
Agreed